### ISSUE TITLE
Whitelist requirement rule which extends another requirement should allow duplicates

### ISSUE THREAD

**Issue by konove** [google/closure-compiler#1232]
Let's say there is a requirement:

requirement: {
  rule_id: 'a'
  whitelist: 'something'
}

If there is a rule which extends it with the same whitelist, the compiler will throw an error:

requirement {
  extends: 'a'
  whitelist: 'something'
}

Result: Invalid requirement. Reason: duplicate whitelist value: something
Expected result: no error should be thrown in this case.

**Comment by concavelenz** [comment]
Presumably you don't control the original? Or what is your use case?

**Comment by konove** [comment]
Yes, the original rule is controlled by another team. At one point the rule was updated and our build started to fail. I think in that case the warning would do just fine.
